website/content/docs/commands/pki/health-check.mdx

```diff
@@ -277,6 +347,29 @@ This health check checks whether unsafe access to APIs (such as `sign-intermedia
 
 This health check verifies if the `If-Modified-Since` header has been added to `passthrough_request_headers` and if `Last-Modified` header has been added to `allowed_response_headers`. This is an informational message if both haven't been configured, or a warning if only one has been configured.
 
+**Remediation Steps**:
+
+Add the missing headers to the `allowed_response_headers` and `passthrough_request_headers` with the command:
+
+```
+vault secrets tune \
+  -passthrough-request-headers="If-Modified-Since" \
+  -allowed-response-headers="Last-Modified" \
+  <mount>
+```
+
+If ACME support is desired, the following command will enable the ACME specific headers as well
+
+```
+vault secrets tune \
+  -passthrough-request-headers="If-Modified-Since" \
+  -allowed-response-headers="Last-Modified" \
+  -allowed-response-headers="Replay-Nonce" \
+  -allowed-response-headers="Link" \
+  -allowed-response-headers="Location" \
+  <mount>
+```
```

Comment:
```suggestion
1. Update `allowed_response_headers` and `passthrough_request_headers` for all
   policies with `vault secrets tune`:
    ```shell-session
    vault secrets tune \
      -passthrough-request-headers="If-Modified-Since" \
      -allowed-response-headers="Last-Modified" \
      <mount>
    ```
1. Update ACME-specific headers with `vault secrets tune` (if you are using ACME):
    ```shell-session
    vault secrets tune \
      -passthrough-request-headers="If-Modified-Since" \
      -allowed-response-headers="Last-Modified" \
      -allowed-response-headers="Replay-Nonce" \
      -allowed-response-headers="Link" \
      -allowed-response-headers="Location" \
      <mount>
    ```
```